Dashboard/citymanage/static/js/dashboard.js

```diff
@@ -0,0 +1,69 @@
+window.onload = function initMap() {
+
+    var stations_locations = new Array([]);
+
+    $(document).ready(function () {
+
+        $.getJSON("/templates/other/stationlocations.json", function (result) {
+            $.each(result, function (i, line) {
+                stations_locations[i] = [
+                    line.No,
+                    line.Name,
+                    line.Latitude,
+                    line.Longitude
+                ];
+            });
+        });
+
+    });
+    var my_place = {
```

Comment:
Do not use names with "my" as prefix, it does not mean anything.  Place is not a expressive name.
Consider "CurrentLocation" or "StartLocation" here.